packages/scene-composer/src/utils/entityModelUtils/SceneLifecycleModule.ts

```diff
@@ -0,0 +1,40 @@
+import { GetSceneCommandOutput } from '@aws-sdk/client-iottwinmaker';
+
+import {
+  LAYER_ROOT_ENTITY_ID,
+  LAYER_ROOT_ENTITY_NAME,
+  SCENE_ROOT_ENTITY_COMPONENT_NAME,
+  DYNAMIC_SCENE_CAPABILITY,
+} from '../../common/entityModelConstants';
+import { getGlobalSettings } from '../../common/GlobalSettings';
+import { KnownSceneProperty } from '../../interfaces';
+import { emptyScene } from '../emptyScene';
+
+import { updateSceneEntityComponent } from './sceneComponent';
+
+export class SceneLifecycleModule {
+  createDynamicScene = async (): Promise<GetSceneCommandOutput | Error> => {
+    try {
+      if (!getGlobalSettings().twinMakerSceneMetadataModule) {
+        return Error('SceneMetadataModule is undefined');
+      }
+      await getGlobalSettings().twinMakerSceneMetadataModule!.createScene({ capabilities: [DYNAMIC_SCENE_CAPABILITY] });
+      const sceneInfo = await getGlobalSettings().twinMakerSceneMetadataModule!.getSceneInfo();
+      const sceneRootEntityId = sceneInfo?.sceneMetadata?.[KnownSceneProperty.SceneRootEntityId];
```

Comment:
let's make an enum for the sceneMetadata keys like this 
```
export enum SceneMetadataMapKeys {
  MATTERPORT_SECRET_ARN = 'MATTERPORT_SECRET_ARN',
  SCENE_ROOT_ENTITY_ID = 'SCENE_ROOT_ENTITY_ID',
}
```
Together with `SceneCapabilities`, they can be put into a new file `sceneModelConstants.ts` under `common` folder

Reply:
Added and pushed!